hazelcast/src/main/java/com/hazelcast/instance/HazelcastInstanceImpl.java

```diff
@@ -102,45 +102,38 @@
     @SuppressWarnings("checkstyle:visibilitymodifier")
     public final Node node;
 
-    final ILogger logger;
+    final ConcurrentMap<String, Object> userContext = new ConcurrentHashMap<String, Object>();
 
+    final ILogger logger;
     final String name;
-
     final ManagementService managementService;
-
     final LifecycleServiceImpl lifecycleService;
-
     final ManagedContext managedContext;
-
-    final ConcurrentMap<String, Object> userContext = new ConcurrentHashMap<String, Object>();
-
     final HazelcastInstanceCacheManager hazelcastCacheManager;
 
     @SuppressWarnings("checkstyle:executablestatementcount")
-    protected HazelcastInstanceImpl(String name, Config config, NodeContext nodeContext)
-            throws Exception {
+    protected HazelcastInstanceImpl(String name, Config config, NodeContext nodeContext) {
         this.name = name;
         this.lifecycleService = new LifecycleServiceImpl(this);
 
         ManagedContext configuredManagedContext = config.getManagedContext();
-        managedContext = new HazelcastManagedContext(this, configuredManagedContext);
+        this.managedContext = new HazelcastManagedContext(this, configuredManagedContext);
 
-        //we are going to copy the user-context map of the Config so that each HazelcastInstance will get its own
-        //user-context map instance instead of having a shared map instance. So changes made to the user-context map
-        //in one HazelcastInstance will not reflect on other the user-context of other HazelcastInstances.
-        userContext.putAll(config.getUserContext());
-        node = createNode(config, nodeContext);
+        // we are going to copy the user-context map of the Config so that each HazelcastInstance will get its own
+        // user-context map instance instead of having a shared map instance. So changes made to the user-context map
+        // in one HazelcastInstance will not reflect on other the user-context of other HazelcastInstances
+        this.userContext.putAll(config.getUserContext());
+        this.node = createNode(config, nodeContext);
 
         try {
-            logger = node.getLogger(getClass().getName());
+            this.logger = node.getLogger(getClass().getName());
```

Comment:
For assignments to fields are prefer to add this as well; because often there is a local variable with the same name (e.g. passed as method/constructor argument).